updater/lib/dependabot/updater/operations/group_update_all_versions.rb

```diff
@@ -20,12 +20,14 @@ module Operations
       class GroupUpdateAllVersions
         include GroupUpdateCreation
 
-        def self.applies_to?(job:)
+        def self.applies_to?(job:) # rubocop:disable Metrics/PerceivedComplexity
           return false if job.updating_a_pull_request?
           if Dependabot::Experiments.enabled?(:grouped_security_updates_disabled) && job.security_updates_only?
             return false
           end
 
+          return true if job.source.directories && job.source.directories.count > 1
```

Comment:
Do we really care that a single entry in `directories` goes through this flow? We could simplify and just have it handle the single directory?

Reply:
Nope, we don't care. I will start refactoring next, operations have basically converged at this point.